plugins/catalog-import/src/components/ImportInfoCard/ImportInfoCard.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InfoCard } from '@backstage/core-components';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
+import { Chip, Typography } from '@material-ui/core';
+import React from 'react';
+import { useImportOptions } from '../ImportOptionsContext';
+
+export const ImportInfoCard = () => {
+  const configApi = useApi(configApiRef);
+  const appTitle = configApi.getOptional('app.title') || 'Backstage';
+  const opts = useImportOptions();
+
+  const integrations = configApi.getConfig('integrations');
+  const hasGithubIntegration = integrations.has('github');
```

Comment:
interestingly (just noting, no action) I think that this can never be false (it inserts a `github.com` one transparently)

Reply:
I think this works as expected as it directly reads from the configuration file, not the integrations API.